src/MarginTrading.SettingsService.SqlRepositories/Repositories/OperationExecutionInfoRepository.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Threading.Tasks;
+using Common;
+using Common.Log;
+using Dapper;
+using MarginTrading.SettingsService.Core.Domain;
+using MarginTrading.SettingsService.Core.Interfaces;
+using MarginTrading.SettingsService.Core.Services;
+using MarginTrading.SettingsService.SqlRepositories.Entities;
+using MarginTrading.SettingsService.StorageInterfaces.Repositories;
+using Microsoft.Extensions.Internal;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace MarginTrading.SettingsService.SqlRepositories.Repositories
+{
+    public class OperationExecutionInfoRepository : IOperationExecutionInfoRepository
+    {
+        private const string TableName = "SettingsServiceExecutionInfo";
+        private const string CreateTableScript = "CREATE TABLE [{0}](" +
+                                                 "[Id] [nvarchar] (64) NOT NULL PRIMARY KEY," +
+                                                 "[LastModified] [datetime] NOT NULL, " +
+                                                 "[OperationName] [nvarchar] (64) NULL, " +
```

Comment:
If we have OperationName in model and in all methods, let's use it )
We can create OID field as PK and add unique index by 2 fields (Id and OperationName).